base/debian/control

```diff
@@ -4,15 +4,33 @@ Section: misc
 Priority: optional
 Build-Depends: config-package-dev,
                debhelper-compat (= 12),
+               clang,
+               pkg-config,
+               meson,
+               libfmt-dev,
```

Comment:
If we're linking these statically it's fine to be in `Build-Depends`, but if we're linking dynamically should these be in `Depends` as well?

Reply:
Done.
Added some of the packages with the -dev suffix because those had the version in the package name and were different between ubuntu20.04, 22.04 and debian. The -dev package depends on the correct one for each platform.